common-libs/common/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# release 2.1.6 (2022-06-22)
+* updated `internal-api-clients`, `tools-common`
```

Comment:
@oleksiipiliugin it is not clean why we update a version of lib could we mention rootcause everywhere in a change log ?